com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/Breakpoint.java

```diff
@@ -70,6 +70,7 @@ public void close() throws Exception {
         subscriptions().forEach(subscription -> {
             subscription.dispose();
         });
```

Comment:
why not to clear subscriptions?

Reply:
This is a reinstall case. Subscriptions doesn't changed.

Reply:
+1
`subscriptions.clear()`

Reply:
Fixed